.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+name: tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    type: [ "opened", "reopened", "synchronize" ]
+  schedule:
+    - cron: '0 12 * * 0'  # run once a week on Sunday
+jobs:
+  tests:
+    strategy:
+      matrix:
+        config:
+        # [Python version, tox env]
+        - ["3.8",   "flake8"]
+        - ["2.7",   "py27"]
+        - ["3.6",   "py36"]
+        - ["3.7",   "py37"]
+    runs-on: ubuntu-latest
+    name: ${{ matrix.config[1] }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.config[0] }}
+    - name: Pip cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('setup.*', 'tox.ini') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox
+    - name: Test
+      run: tox -e ${{ matrix.config[1] }}
+    - name: Coverage
```

Comment:
If we intend not to have coverage in this package, we can omit this part.

Reply:
I added coverage from your example in https://github.com/gocept/js.jquery_datatables_plugins/pull/2